src/ui/routing/zcl_abapgit_services_repo.clas.abap

```diff
@@ -180,22 +174,6 @@ CLASS zcl_abapgit_services_repo IMPLEMENTATION.
   ENDMETHOD.
 
 
-  METHOD check_and_create_package.
-
-    IF zcl_abapgit_factory=>get_sap_package( iv_package )->exists( ) = abap_false.
-      " Check if any package is included in remote
-      READ TABLE it_remote TRANSPORTING NO FIELDS
-        WITH KEY file
-        COMPONENTS filename = zcl_abapgit_filename_logic=>c_package_file.
-      IF sy-subrc <> 0.
-        " If not, prompt to create it
-        create_package( iv_package ).
```

Comment:
how about, here instead of showing the popup, throw an exception? @mbtools

Reply:
Don't have time to look at it now. There should not be a popup (b-case) but an error to fallback to the "new repo" form.